Delicate-Butterflies/Bangazon-Site
# A user can complete an order

Related Issue: #2 

Description: User when views their cart(open order), they can add payment and complete the order

Chai/Mocha Tests File: None

PR Testing Steps:
- login as one of the users from faker (use password: abcd)
- go to shopping cart
 - should display No orders if none or once you have finished payment.
 - should display all products with total
- Click complete order 
- should by default select 1st payment method or the one you select
- should take you to Confirmation page. 

Check list before you make a PR:
